src/resources/material.js

```diff
@@ -156,7 +156,12 @@ class MaterialHandler {
         const placeholder = PLACEHOLDER_MAP[parameterName];
         const texture = this._placeholderTextures[placeholder];
 
-        materialAsset.resource[parameterName] = texture;
+        return texture;
+    }
+
+    _assignPlaceholderTexture(parameterName, materialAsset) {
```

Comment:
descriptive usage comment could be useful here

Reply:
I'll refactor the comment

Reply:
done